sky/shell/platform/mojo/BUILD.gn

```diff
@@ -44,4 +49,11 @@ mojo_native_application("mojo") {
     "//third_party/icu",
     "//url",
   ]
+
+  if (is_android) {
+    ldflags = [
+      "-lGLESv2",
```

Comment:
But those symbols won't do what you want in a mojo build.  If you're running in mojo you need to use the MGL-provided symbols only.  Is that file in the skia repo?